web/README.md

```diff
@@ -12,8 +12,36 @@ The Authentication Playground was designed to provide an intuitive and functiona
 This application is built using  [React.JS](https://reactjs.org) on the frontend, themed with the [Material-UI](https://material-ui.com) library and uses [Go](https://golang.org) for the backend.
 
 ## Quickstart
+First clone the repository using the following command:
+
+```bash
+$ git clone https://github.com/googleinterns/oauth2l-web
+```
+Then go into the `web` directory in your `oauth2l-web` directory.
+```bash
+$ cd oauth2l-web

```

Comment:
Do you mean cd oauth2l-web/web ?

Reply:
Yep that is what I meant! Thanks for catching that.